Makefile

```diff
@@ -113,7 +114,7 @@ install: generate cert-mgr ## Install rukpak
 run: build-container kind-cluster kind-load install ## Build image, stop/start a local kind cluster, and run operator in that cluster
 
 cert-mgr: ## Install the certification manager
-	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/$(CERT_MGR_VERSION)/cert-manager.yaml
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MGR_VERSION}/cert-manager.yaml
```

Comment:
Mainly curious - any idea on why this change was necessary?

Reply:
I was getting bugs running this locally. I assume it's something related to `$()` being used to launch a subshell but that's a pure guess because I'm a bash noob tbh. I can change it back.